### DISCUSSION TITLE
Has geomview support been removed?

### DISCUSSION THREAD

**Discussion by insert-username** [CGAL/cgal#6171] [Q&A] [answered]
Hi,
I've just started playing with this library so please excuse me if this is a stupid question, but I've been trying to get the GeomView example working and I noticed that Installation/include/CGAL/config.h has CGAL_USE_GEOMVIEW defines commented out since 5.2 at least. Is this a deliberate change? If so is there a recommended way to include GeomView support when installing?
(Installed from source using the git repo, mkdir build, cmake, make install etc.)
Thanks!

**Comment by lrineau** [comment, marked as answer]
The Geomview support in CGAL has not been tested for a long time. We decided to stop supporting it.

That was in PR #5299. The change was actually introduced in the release cycle for CGSAL-5.3, but was backported to CGAL-5.1.3 and CGAL-5.2.1. It should have been properly announced in the changelog, and not backported. It was my fault: I decided to drop the support for Geomview in all our release branches to simplify our CI infrastructure.

**Reply by lrineau** [reply]
Actually, the documentation and examples about Geomview should be removed as well.

**Reply by gdamiand** [reply]
And replaced by examples based on basic viewer.

**Reply by insert-username** [reply]
Ah ok, thanks for letting me know. It was a little confusing trying to follow the documentation when the feature had been removed :p . I take it "basic viewer" is an alternative tool?
